core/src/main/java/org/elasticsearch/monitor/fs/FsService.java

```diff
@@ -33,7 +33,7 @@
  */
 public class FsService extends AbstractComponent {
 
-    private final FsProbe probe;
+    private FsProbe probe;
```

Comment:
I think we should use an ExtensionPoint instead and be able to register the FsProbe to use. I'll have a look at it but it would be nice to have something similar to `Transport` were we can register different transport modes but use one specifically.